src/views/AddItem.jsx

```diff
@@ -1,3 +1,87 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	// sets initial default values in form fields and deconstruct form field variables
+	const initialState = { itemName: '', estimate: '7' };
+	const [formData, setFormData] = useState(initialState);
+	const { itemName, estimate } = formData;
+
+	// for now, we retrieve the token for the test list from local storage;
+	const token = window.localStorage.getItem('tcl-shopping-list-token');
+
+	// updates state based on user input for multiple form fields
+	const formHandler = (e) => {
+		const name = e.target.name;
+		const value = e.target.value;
+		setFormData((formData) => ({ ...formData, [name]: value }));
+	};
```

Comment:
Nifty generalized `formHandler`! I like how you used the `name` property on the inputs to determine which property was being updated. 👍